Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -864,7 +1088,7 @@ def main():
                 raise NotImplementedError(f"Command {command} is not implemented")
```

Comment:
There's another f-string here that needs to be changed.

Reply:
Removed